src/core/tests/type_prop/shuffle_channels.cpp

```diff
@@ -30,14 +32,16 @@ TEST(type_prop, shuffle_channels_basic_4D) {
 }
 
 TEST(type_prop, shuffle_channels_dynamic_4D) {
-    const auto data_input_shape = PartialShape{Dimension::dynamic(), Dimension(3, 9), 4, Dimension(4, 15)};
+    auto data_input_shape = PartialShape{Dimension::dynamic(), Dimension(3, 9), 4, Dimension(4, 15)};
+    set_shape_labels(data_input_shape, 10);
     const auto data = make_shared<op::Parameter>(element::f32, data_input_shape);
     const auto axis = 1;
     const auto group = 3;
     const auto shuffle_channels = make_shared<op::v0::ShuffleChannels>(data, axis, group);
 
     EXPECT_EQ(shuffle_channels->get_element_type(), element::f32);
     EXPECT_EQ(shuffle_channels->get_output_partial_shape(0), data_input_shape);
+    EXPECT_THAT(get_shape_labels(shuffle_channels->get_output_partial_shape(0)), ElementsAre(10, 11, 12, 13));
```

Comment:
Actually, the data is being reshuffled, so channel dimension must have no label. please clear it in the shape inference. Please check for other operations too

Reply:
Changed.